modules/detect_target/detect_target.py

```diff
@@ -84,7 +83,9 @@ def run(self, data: image_and_time.ImageAndTime) -> "tuple[bool, np.ndarray | No
 
             self.__counter += 1
 
-        # TODO: Change this to DetectionsAndTime
-        return True, image_annotated
+        if self.__show_annotations:
+            cv2.imshow("Annotated Image", image_annotated)
+
+        return True, detections
```

Comment:
Does it work on your end? I don't think `cv2.imshow` is process safe. Maybe consider passing `image_annotated` to the main process and display it from there instead of in worker process.

Reply:
It probably isn't threadsafe, but since only a single process will be using it, it should be fine. See the Autonomy bootcamp for an example of where it works without issue.

Reply:
I tried it on my machine and the window freezes on startup if placed in worker process. Not sure if it's the case for @Ethan118 too.

Reply:
Did a little test and when `cv2.imshow` was placed in worker process, the window freezes at start up my machine. For where it's currently placed, it works fine. Not sure if it's the same case with @Ethan118